src/components/Input/styles/InputRevealPasswordWrapper.js

```diff
@@ -14,8 +14,8 @@ const InputRevealPasswordContainer = styled.div`
 
   input[type='password'],
   input[type='text'] {
-    padding: ${get('spacing.medium')} ${get('spacing.extraLarge')}
-      ${get('spacing.medium')} ${get('spacing.medium')} !important;
+    padding-right: ${get('spacing.extraLarge')};
```

Comment:
Doesn't this need to be `!important` too?

Reply:
seem to work fine without it.

Reply:
seem to work fine without it.